src/mqt/bench/qiskit_helper.py

```diff
@@ -192,9 +160,9 @@ def get_native_gates_level(
 @overload
 def get_mapped_level(
     qc: QuantumCircuit,
-    gate_set_name: str,
+    gate_set: list[str],
```

Comment:
To generate as many test circuits as possible it could be nice to leave the option available. I agree however that in a real world scenario it does not make too much sense to map a foreign gate set onto a device that does not support it.